docs/api/1.x/accounts.rst

```diff
@@ -294,3 +294,166 @@ Or delete some account:
 ::
 
     $ http DELETE http://localhost:8888/v1/accounts/sam-body --auth admin:s3cr3t
+
+
+
+.. _accounts-validate:
+
+Validate accounts
+=================
+
+If the ``account validation`` option in :ref:`the settings
+<settings-accounts>` has been enabled, account IDs need to be valid email
+addresses: they need to match the regexp in the
+``account_validation.email_regexp`` setting.
+
+.. sourcecode:: bash
+
+    $ echo '{"data": {"id": "bob@example.com", "password": "azerty123"}}' | http POST http://localhost:8888/v1/accounts --verbose
+
+If the user was created, an email will be sent to the user with a link to a
+form with the activation key. This form should POST this activation key to the
+``validate`` endpoint.
```

Comment:
To be restful, we want this step to be a POST, this would also prevent developer to point their user to a JSON API endpoint rather than to an interface.